<?xml version="1.0"?>
<ui version="4.0">
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QGroupBox" name="groupScan">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Scan geometry and parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="5" column="0">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QFormLayout" name="formScanParameters">
             <item row="0" column="0">
              <widget class="QSpinBox" name="spinScanNumber">
               <property name="toolTip">
                <string>The number of scans to acquire. Note that these are saved independently, see Repeat/Averaging section</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelScanNumber">
               <property name="toolTip">
                <string>The number of scans to acquire. Note that these are saved independently, see Repeat/Averaging section</string>
               </property>
               <property name="text">
                <string>Scan number</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QFormLayout" name="formX">
             <item row="0" column="0">
              <widget class="QDoubleSpinBox" name="spinROIWidth">
               <property name="toolTip">
                <string>The length of the B-scan in physical units</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelROIWidth">
               <property name="toolTip">
                <string>The length of the B-scan in physical units</string>
               </property>
               <property name="text">
                <string>X-axis ROI width</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinACount">
               <property name="toolTip">
                <string>The number of A-scans in a B-scan</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="suffix">
                <string> px</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1200</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="labelACount">
               <property name="toolTip">
                <string>The number of A-scans in a B-scan</string>
               </property>
               <property name="text">
                <string>A-scan count</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QDoubleSpinBox" name="spinFastAxisSpacing">
               <property name="toolTip">
                <string>The spacing between adjacent A-scans along the fast axis</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="suffix">
                <string> um</string>
               </property>
               <property name="decimals">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="labelFastAxisSpacing">
               <property name="toolTip">
                <string>The spacing between adjacent A-scans along the fast axis</string>
               </property>
               <property name="text">
                <string>Fast axis spacing</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="checkEqualAspect">
             <property name="toolTip">
              <string>Forces X and Y scan spacing to be equal</string>
             </property>
             <property name="text">
              <string>Equal aspect scan</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <layout class="QFormLayout" name="formY">
             <item row="0" column="1">
              <widget class="QLabel" name="labelROIHeight">
               <property name="toolTip">
                <string>The width of the C-scan in physical units</string>
               </property>
               <property name="text">
                <string>Y-axis ROI height</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="labelBcount">
               <property name="toolTip">
                <string>The number of B-scans to acquire</string>
               </property>
               <property name="text">
                <string>B-scan count</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QDoubleSpinBox" name="spinSlowAxisSpacing">
               <property name="toolTip">
                <string>The distance between adjacent B-scans in physical units</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="suffix">
                <string> um</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="labelSlowAxisSpacing">
               <property name="toolTip">
                <string>The distance between adjacent B-scans in physical units</string>
               </property>
               <property name="text">
                <string>Slow axis spacing</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinBcount">
               <property name="toolTip">
                <string>The number of B-scans to acquire</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="suffix">
                <string> px</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1200</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QDoubleSpinBox" name="spinROIHeight">
               <property name="toolTip">
                <string>The width of the C-scan in physical units</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>4</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
 <pixmapfunction></pixmapfunction>
 <connections/>
</ui>
